packages/ckeditor5-ui/src/bindings/addkeyboardhandlingforgrid.ts

```diff
@@ -84,4 +82,52 @@ export default function addKeyboardHandlingForGrid(
 			evt.preventDefault();
 		};
 	}
+
+	// Function returning a right index relatively current index.
+	//
+	// before: [ ][x][ ]	after: [ ][ ][x]
+	//         [ ][ ][ ]	       [ ][ ][ ]
+	//         [ ]      	       [ ]
+	// index = 1            index = 2
+	//
+	// If current index is at the right conner, function return first index of next row.
+	//
+	// before: [ ][ ][x]	after: [ ][ ][ ]
+	//         [ ][ ][ ]	       [x][ ][ ]
+	//         [ ]      	       [ ]
+	// index = 2            index = 3
+	//
+	// @param {number} [elementIndex] Number of current index.
+	// @param {module:ui/viewcollection~ViewCollection} [gridItems] A collection of grid items.
+	function getRightElementIndex( elementIndex: number, gridItems: ViewCollection ) {
+		if ( elementIndex === gridItems.length - 1 ) {
+			return 0;
+		} else {
+			return elementIndex + 1;
+		}
+	}
```

Comment:
It's usually a good practise to only pass what is necessary to the function - it makes it easier to understand, refactor etc. Here, if you look carefully, we're passing `gridItems` object, but the function actually only uses its length - so it makes sense to pass just that - the length of the collection, and name it e.g. `collectionLength`. It will show that we don't do any operations on the collection inside the function.